src/games/strategy/triplea/ui/BattleDisplay.java

```diff
@@ -362,28 +362,28 @@ private Territory getSubmerge(final String message) {
     final Territory[] retreatTo = new Territory[1];
     final CountDownLatch latch = new CountDownLatch(1);
     final Action action = SwingAction.of("Submerge Subs?", e -> {
-      final String ok = "Submerge";
```

Comment:
It's how the merge conflicts got resolved. Looks like only indentation was changed.